provisioning/modules/chromedriver/manifests/init.pp

```diff
@@ -0,0 +1,37 @@
+# install chromedriver for functional tests
+# we ship our local copy of chromedriver
+# because latest versions are getting recurring errors on
+# test/functional/features/login.feature#Then I should see the fancy interstitial
+
+class chromedriver (
+  $release       = '1.0_beta1',
+  $chromedriver  = 'chromedriver_2.27_linux64.zip',
+  $google_chrome = 'google-chrome-stable_54.0.2840.100-1_amd64.deb',
+) {
+
+  exec { 'fetch_chromedriver':
+    command => "/usr/bin/wget https://github.com/pixelated/pixelated-user-agent/releases/download/${release}/${chromedriver}",
+    cwd     => '/var/tmp',
+    creates => "/var/tmp/${chromedriver}",
+  }
+
+  exec { 'fetch_google_chrome':
+    command => "/usr/bin/wget https://github.com/pixelated/pixelated-user-agent/releases/download/${release}/${google_chrome}",
+    cwd     => '/var/tmp',
+    creates => "/var/tmp/${google_chrome}",
+  }
+
+  exec { 'unpack_chromedriver':
+    command => "/usr/bin/unzip ${chromedriver} -d /usr/local/bin/",
+    cwd     => '/var/tmp/',
+    creates => '/usr/local/bin/chromedriver',
```

Comment:
This resource depends on Exec['fetch_chromedriver'] so pls add a `require` here.